tests/end_to_end/cypress/support/index.ts

```diff
@@ -0,0 +1,10 @@
+import './commands'
+
+beforeEach(() => {
+  cy.dockerStart();
+});
+
+afterEach(() => {
+// Added this time to delay the end of video recording
+  cy.wait(5000)
```

Comment:
Why?

Reply:
It was during the "tests", I forgot to take it off.